### ISSUE TITLE
Wrong appointment times at Zoom

### ISSUE THREAD

**Issue by RainerZufall9393** [calcom/cal.com#7844]
The booked appointments are displayed with wrong time at Zoom.

- At 1. I had an appointment today at 19:00 - 20:00. In Zoom, however, it says 19:30 - 20:30.
- In 2. I created a test appointment for today 20:30 - 22:00. However, Zoom says 20:21 - 21:51.

<img width="992" alt="SCR-20230320-rsbt-2" src="https://user-images.githubusercontent.com/126205884/226445720-79015ed7-43b4-440c-95ab-00ab44256819.png">

**Comment by emrysal** [comment]
Hi @RainerZufall9393 - Merged a fix for this recently https://github.com/calcom/cal.com/pull/7820 which will be deployed in the next hour, would you be willing to confirm the bug is fixed after this deployment?

**Comment by RainerZufall9393** [comment]
Yes I will check again after the release of the update and report then. @emrysal

**Comment by RainerZufall9393** [comment]
Seems to work now with 2.7.2 update again without problems. Have it now 2x tested again. 👍 @emrysal

**Comment by emrysal** [comment]
Great 🥳 - Glad it works now - that was a proper rabbit hole of an API investigation. Thanks for verifying 🙏

**Comment by RainerZufall9393** [comment]
You're very welcome. Thanks for the fix! 🙂 @emrysal

**Comment by PeerRich** [comment]
epic!
